### ISSUE TITLE
Screenshot capturing failed for kali app in windows

### ISSUE THREAD

**Issue by BhavarthMtech** [michenriksen/aquatone#232]
I am having issue capturing screenshot when I run cat abc.txt | ./aquatone -chrome-path chrom/chromium -out abc
I created one directory called aquatone. In that abc.txt file exists and all the necessary files. 
I am running the windows kali linux application to execute.
Below error is generated:
screenshot failed: exit status 1

**Comment by theblackturtle** [comment]
I also got this error too. This is output when turn debug on
```
[agent:url_screenshotter] Error: signal: killed
http://domain.com: screenshot timed out
```

**Comment by osamahamad** [comment]
i got the same error in ubuntu 18

**Comment by UWA-Nigel** [comment]
I'm also seeing screenshot timeout failure, running this from Win10 in PowerShell 7
`cat ..\domains.txt | .\aquatone.exe -out .\domainsOUT\ -chrome-path ..\chromium-761240\chrome.exe`

**Comment by UWA-Nigel** [comment]
Came back to say that I got the screenshot working after reading some of the closed issues!

`cat ..\domains.txt | .\aquatone.exe -out C:\Tools\Aquatone\domainsOUT\ -chrome-path C:\Tools\chromium-761240\chrome.exe`

Almost seems as if the compiled .exe script doesn't look at the current working directory?

**Comment by glafarge** [comment]
Chrome trying to save screenshot to a relative path that doesn't exists (under path/to/chrome/screenshots next to executable). 
Maybe the pull-request #243 will help you fixing the issue : the path now refering to `destDir/screenshots` folder.

**Comment by alandixon** [comment]
I agree. Chrome, or Aquatone (?) seems to be confused about current dirs.
I fixed this by 
- Making sure the proposed output dir already exists
- Giving Aquatone an absolute path with -out
